src/test/regress/spec/isolation_replicate_reference_tables_to_coordinator.spec

```diff
@@ -123,14 +122,25 @@ step "deadlock-checker-call"
   SELECT check_distributed_deadlocks();
 }
 
+
+// adding node in setup stage prevents getting a gpid with proper nodeid
```

Comment:
prevents, because session 1 seems to use the same backend as the step 1. So, we get gpid that is assigned before coordinator is added to the coordinator.

Though, this also didn't help, I think it is because cache invalidation is not working properly on the  isolation tests. Locally, this works fine